central/scannerv4definitions/repomapping/updater_test.go

```diff
@@ -0,0 +1,58 @@
+package repomapping
+
+import (
+	"archive/zip"
+	"net/http"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/central/scannerdefinitions/file"
+	"github.com/stackrox/rox/pkg/fileutils"
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	defURL = "https://storage.googleapis.com/scanner-v4-test/redhat-repository-mappings/"
```

Comment:
I recommend not hard-coding this. Also, the default should not be `scanner-v4-test`, but the production bucket name we are planning to have. If we don't know the name yet, create a task to update from `scanner-v4-test`.